src/github.com/matrix-org/dendrite/clientapi/routing/routing.go

```diff
@@ -406,4 +406,31 @@ func Setup(
 			}}
 		}),
 	).Methods(http.MethodGet, http.MethodOptions)
+
+	r0mux.Handle("/user/{userId}/rooms/{roomId}/tags",
+		common.MakeAuthAPI("get_tag", authData, func(req *http.Request, device *authtypes.Device) util.JSONResponse {
+			// When a GET tag request is done, obtain all data of user from here and provide a JSON response.
```

Comment:
These comments should go to the GetTag function.
Since we are only routing the request go to GetTag (at this point).

Reply:
Alright